first-setup

pPitohu opened the pull request.

pPitohu committed:
first-setup

psychobug420 commented on src/App.tsx:
```diff
@@ -1,3 +1,7 @@
+import React, { useState, useEffect } from 'react';
 
-export default () => <div className="App">EAZY LEARN</div>;
+const App: React.FC = function () {
```
I would prefer function declaration or using arrow function, I don't understand why you used function expression, you can try to explain me your choice

pPitohu replied:
Whoops, miss that, i was thinking i'm writing an arrow function

psychobug420 commented on .vscode/settings.json:
```diff
@@ -0,0 +1,8 @@
+{
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint",
```
I would prefer Prettier than eslint to format code, and it wasn't part of ticket to setup formatters, next time ask me about such questions if ticket too short

psychobug420 commented on .eslintrc.json:
```diff
@@ -1,17 +1,48 @@
 {
-  "env": {
```
This file is okay, in future we would improve eslint rules more

psychobug420 commented on src/index.tsx:
```diff
@@ -3,11 +3,11 @@ import ReactDOM from 'react-dom';
 import App from './App';
 
-const funct = (): any => "hello"
+const funct = (): any => 'hello';
```
Please remove this function I forget to remove it, it was just to check eslint

pPitohu committed:
structured file

husky-test

husky-test

husky-test

husky-test

husky-test

husky-test-via-lint-staged

husky-test-via-lint-staged

husky-test-via-lint-staged

husky-test-via-lint-staged

psychobug420 commented on .gitignore:
```diff
@@ -1,5 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
+/.vscode
```
remove /.vscode from .gitingore every developers in our team use it)

psychobug420 commented on .husky/pre-commit:
```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```
That file is also okay, but in future we would improve this

pPitohu committed:
removed .vscode out of .gitignore
